sdks/java/core/src/main/java/org/apache/beam/sdk/values/Row.java

```diff
@@ -208,6 +210,26 @@ public Boolean getBoolean(String fieldName) {
     return getMap(getSchema().indexOf(fieldName));
   }
 
+  /** Helper method to get an enum field value. */
+  public EnumerationValue getEnumValue(String fieldValue) {
```

Comment:
Same remark. Are they first class citizens to merit extra methods?

Reply:
See above (also keep in mind that we want to make DATETIME a logical type as well). However I don't feel strongly one way or the other about this.

Reply:
Mmm.. yes. But we should keep it to the basic well known types then. Enum, DateTime they make sense. But the OneOf, is not so common.

Reply:
IMO unions are pretty basic. However removed these methods for now, as it's not difficult to simply access as a logical type.